imports/chrome/segments.go

```diff
@@ -0,0 +1,66 @@
+package chrome
+
+import (
+	"gorm.io/gorm"
+)
+
+type Segment struct {
+	gorm.Model
+	Name  string `gorm:"uniqueIndex:chrome_segments_key"`
+	URLID int    `gorm:"uniqueIndex:chrome_segments_key"`
+	URL   URL
+}
+
+func (Segment) TableName() string {
+	return tablePrefix + "segments"
+}
+
+func (s Segment) Conditions() map[string]interface{} {
+	return map[string]interface{}{
+		"url_id": s.URLID,
+	}
+}
+
+func (p *chrome) importSegments(db *gorm.DB) error {
+	selection := "id, name, url_id"
+
+	var segments []Segment
+	err := db.
+		Raw("select "+selection+" from segments order by id limit ?", dbRowSelectLimit).
```

Comment:
why not ORM?

Reply:
You have to define whole scheme (and duplicate code that way) to use ORM